src/bridge.ts

```diff
@@ -705,12 +706,16 @@ export class Bridge {
             const now = Date.now();
             for (const [key, entry] of this.intents.entries()) {
                 // Do not delete intents that sync.
-                if (this.eeEventBroker?.shouldAvoidCull(entry.intent)) {
```

Comment:
In the previous version of the code, we ended up never culling any Intents because of an erronous `return` statement (I'm good at those).

This change means that Intent jobs are cleaned up if they aren't used after INTENT_CULL_EVICT_AFTER_MS and the eeEventBroker is instantiated AND needs the user kept alive (e.g. they are the transport for encrypted messages in a room)